### ISSUE TITLE
Replace XCTest assertions with Nimble

### ISSUE THREAD

**Issue by fuzza** [fuzza/SwiftyJanet#15]
Using Nimble could drastically reduce boilerplate for assertions. I think we could start replacing some existing XCTest assertions with Nimble's ones.
There is no need to do it in scope of single PR, let's just reference this issue in all corresponding PRs

**Comment by demalex** [comment]
@fuzza Does `swiftlint` rule [Nimble operator](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/NimbleOperatorRule.swift) is useful for us?

**Comment by fuzza** [comment]
I think so. Nimble has a lot of overloaded operators that allow you to shorten syntax. For instance, you can use 
```swift
expect(var) == anotherVar
```
instead of
```swift
expect(var).to(equal(anotherVar))
```
Do you have any concerns with this rule?

**Comment by demalex** [comment]
Yeah, I got it. I am just asking which way is preferred for us. I am totally good with shorter syntax.
